front-end/src/services/ProjectDataService.js

```diff
@@ -1,27 +1,27 @@
-import http from "../http-common/http-common.projects";
+import http from "../http-common/http-common";
 
 
 // This class contains all the functions that make API calls and return info from the calls.
 class ProjectDataService {
     getAllProjects() {
-        return http.get('/');
+        return http.get('/api/v1/projects/');
```

Comment:
we can tackle this later, but we should make a separate http-common for projects and transcripts so we can avoid directly depending on this API.

Reply:
Yep maybe make a different common file for each use case? we can discuss that